integreat_compass/cms/templates/index.html

```diff
@@ -4,15 +4,114 @@
     {% translate "Home" %}
 {% endblock title %}
 {% block content %}
-    <div class="absolute top-0 right-0 bottom-0 left-0 flex justify-center items-center bg-background-900 z-1">
-        <div class="flex flex-col justify-center gap-4 text-center">
-            <h1 class="font-bold text-5xl drop-shadow-md mb-12 text-white">{% translate "Arriving, reimagined." %}</h1>
-            <a href="#"
-               class="bg-primary hover:bg-red-500 text-white font-bold text-3xl py-4 px-4 rounded-2xl">{% translate "Discover Integreat Compass" %}</a>
-            <a href="{% url 'cms:protected:new_offer' %}"
-               class="bg-white hover:bg-gray-200 font-bold text-3xl py-4 px-4 rounded-2xl">{% translate "Add new offer" %}</a>
+    <div class="flex container mx-auto h-[calc(100vh-6rem)]">
```

Comment:
Oh, you're right. I remember that at one point of the process I added it out of necessity, but since then I made some change that has made this piece of code obsolete